TombEngine/Objects/TR3/Entity/tr3_trex.cpp

```diff
@@ -168,18 +169,20 @@ namespace TEN::Entities::Creatures::TR3
 			case TREX_STATE_ATTACK:
 				if (item->TouchBits.Test(TRexAttackJoints))
 				{
-					item->Animation.TargetState = TREX_STATE_KILL;
-
-					DoDamage(LaraItem, 1500);
-					LaraTRexDeath(item, LaraItem);
+					CreatureKill(item, TREX_ANIM_KILL, LEA_TREX_DEATH, TREX_STATE_KILL, LS_DEATH);
+					Camera.targetDistance = BLOCK(3);
 				}
 
-				item->Animation.RequiredState = TREX_STATE_WALK_FORWARD;
+				break;
+
+			case TREX_STATE_KILL:
+				headYRot = 0;
+				creature->MaxTurn = 0;
 				break;
 			}
 		}
 
-		CreatureJoint(item, 0, headYRot * 2);
+		CreatureJoint(item, 0, headYRot);
```

Comment:
Why multiplier removed?

Reply:
Why was it there?

The TRex had a ability to look at Lara while she is in front of it. However, originally when Lara was in front of it, the Raptor was not looking at Lara, but it turned its head and looked behind him. (In the oposite direction of Lara) this was reported and happened that the cause was that *2 multiplication. 

The other enemies are not using it like that and when I removed it, the TRex started to look at Lara as was expected. 

My theory is that maybe the TR1 mesh may have the transform rotated and Core for some reason decided correct it with that.